wagtail_draftail_anchors/static_src/wagtaildraftailanchors/js/wagtail_draftail_anchor.js

```diff
@@ -152,26 +157,74 @@ class UneditableAnchorDecorator extends React.Component {
     this.setState({ showTooltipAt: null });
   }
 
+  getBlock(editorState){
+    const block_key = editorState.getSelection().getFocusKey();
+    return this.props.contentState.getBlockForKey(block_key);
+  }
+
+  getData(editorState) {
+    const block = this.getBlock(editorState);
+    return block.getData();
+  }
+
+  onRemove(e) {
+    e.preventDefault();
+    e.stopPropagation();
+    this.setState({ showTooltipAt: null });
```

Comment:
This doesn't actually remove it, which feels confusing.

Reply:
Thx, I will check and let you know soon

Reply:
I have fixed. Now onRemove sets anchor to null, which means that slugified header text will be set as id. Also I have renamed button to 'Reset'.

Reply:
@jacobtoppm will you have time to have a look?